pkg/config/config.go

```diff
@@ -36,6 +37,8 @@ type Feed struct {
 	Format model.Format `toml:"format"`
 	// Custom image to use
 	CoverArt string `toml:"cover_art"`
+	// Only download episodes that match this regexp (defaults to matching anything)
+	OnlyMatching Pattern `toml:"only_matching"`
```

Comment:
1. Can we drop `Pattern` and use string instead?
```go
if config.Filters.Title != "" {
    if !regexp.MatchString(config.Filters.Title, episode.Title) { ... }
}
```

2. Let's add a separate structure for the filters:
```toml:
[feeds]
  [feeds.XYZ]
  ...
  quality = "low"
  filters = { title = "regex for title here" }
```
```go
type Filters struct {
	Title string `toml:"title"`
        // More filters to be added here.
}
```

Reply:
Done